python/sbp/client/drivers/network_drivers.py

```diff
@@ -76,9 +76,9 @@ def read(self, size):
             return data
         except socket.timeout:
             self._connect()
-        except socket.error as e:
+        except socket.error as socket_exception:
```

Comment:
Maybe `socket_error`. Name `socket_exception` seems to me would cover all the socket exception classes.

Reply:
fixed